pgx_async/src/pgx_async.ml

```diff
@@ -87,6 +90,12 @@ module Thread = struct
     type 'a monad = 'a t
     type 'a t = 'a Sequencer.t
 
+    let sexp_of_t _ = Sexplib0.Sexp.Atom "<opaque>"
+    let t_of_sexp sexp =
+      Sexplib0.Sexp_conv.of_sexp_error
+        "opaque_of_sexp: cannot convert opaque values"
+        sexp
```

Comment:
Can we just not provide `t_of_sexp`?

Reply:
We should remove this i think. `sexp_of` might be the only converted we need since we can't convert it back from sexp anyways. I'll remove the `t_of_sexp` converters

Reply:
Removed the t_of_sexp